src/components/CertificateVerifyBlock/DetailedCertificateVerifyBlock.test.js

```diff
@@ -1,7 +1,6 @@
-import React from "react";
```

Comment:
I have the feelinds that the test in this component and in `UnverfiedView` are quite similar. Do you think the same ? should we remove one of the tests ?

Reply:
quite similar but not exact..as it renders all the status success and failure at the same time. while the other one only displays the failure statuses. I think we can keep.